### PR TITLE
update file/component includes for Forestry admin section - WIP

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

updating code in Forestry admin components after it was copy and pasted
this is WIP. I want to check it's working locally before merging.

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-90

File: app/frontend/src/components/forestrysectoropscreening/admin/AdminReviewSubmission.vue [hunk 1]
```diff
@@ -41,21 +41,21 @@
...
+    Step1,
     Step2,
     Step3,
-    Step4,
-    Step5
+    Step4
   },
   computed: {
-    ...mapGetters('form', ['gettingForm', 'attestation', 'submissionComplete']),
+    ...mapGetters('ForestrySectorOpScreeningForm', ['gettingForm', 'attestation', 'submissionComplete']),
```
Comment (jujaga) [inline app/frontend/src/components/forestrysectoropscreening/admin/AdminReviewSubmission.vue:58]: This needs to be `forestrySectorOpScreeningForm` (note the lowercase f. This namespace is loaded on the top level forestrysectoropscreening.vue file and should match that.

Comment (TimCsaky) [reply app/frontend/src/components/forestrysectoropscreening/admin/AdminReviewSubmission.vue:58]: oh ok. thanks!

**Comment by loneil** [comment]
@TimCsaky should rebase in my changes from the public side then we can test out this form on this PR then merege.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-90

**Pull request merged**